Std/Data/Rat/Lemmas.lean

```diff
@@ -16,6 +16,9 @@ namespace Rat
 @[simp] theorem one_num : (1 : Rat).num = 1 := rfl
 @[simp] theorem one_den : (1 : Rat).den = 1 := rfl
 
+@[simp] theorem mk_den_one {r : Int} :
+    (⟨r, 1, Nat.one_ne_zero, Nat.coprime_one_right _⟩ : Rat) = r := rfl
```

Comment:
Coercion should be on the rhs.
```suggestion
@[simp] theorem mk_den_one (r : Int) : 
  ⟨r, 1, Nat.one_ne_zero, Nat.coprime_one_right _⟩ = (r : Rat) := rfl
```

Reply:
I accepted all but the indent, since I think that 4 spaces is standard for pre-proof.